csvqb/csvqb/models/cube/qb/components/unit.py

```diff
@@ -141,6 +141,20 @@ def new_units_from_data(data: PandasDataTypes) -> "QbMultiUnits":
             [NewQbUnit(u) for u in set(pandas_input_to_columnar_str(data))]
         )
 
+    @staticmethod
+    def existing_units_from_data(
+        data: PandasDataTypes, csvw_column_name: str, csv_column_uri_template: str
+    ) -> "QbMultiUnits":
+        columnar_data = pandas_input_to_columnar_str(data)
+        return QbMultiUnits(
+            [
+                ExistingQbUnit(
+                    uritemplate.expand(csv_column_uri_template, {csvw_column_name: m})
+                )
+                for m in sorted(set(columnar_data))
```

Comment:
Sorting necessary?

Reply:
Strictly, no, however I prefer idempotent outputs from processes like these. So some form of consistent ordering is necessary.